app/gateway/2.7.x/configure/auth/oidc-okta.md

```diff
@@ -9,84 +9,209 @@ For information about configuring OIDC using Okta as an Identity provider
+{{site.base_gateway}} whose access you want to protect with Okta. For this
+guide, assume the route is in the default workspace.
+* If using {{site.base_gateway}} locally, you need Internet access.
+* Any network access control to your Kong node must allow traffic to and from Okta, the upstream service, and the client.
 
-    <img src="https://doc-assets.konghq.com/0.35/plugins/oidc-okta/06-name-auth.png">
+    Because OpenID Connect deals with user credentials, all transactions should take place over HTTPS.
+    Although user passwords for third party identity providers are only submitted to
+    those providers and not Kong, authentication tokens grant access to a subset of
+    user account data and protected APIs, and should be secured. As such, you should
+    make Kong's proxy available via a fully-qualified domain name and
+    [add a certificate](/gateway/{{page.kong_version}}/admin-api/#certificate-object) for it.
 
-6. Click **Save** and view your Authorization Server Settings.
+### Configure Okta
 
-    <img src="https://doc-assets.konghq.com/0.35/plugins/oidc-okta/07-auth-server-settings.png">
+1. [Register](https://developer.okta.com/docs/guides/add-an-external-idp/openidconnect/register-app-in-okta/) the application you are using Kong to proxy.
+1. From the left menu, select **Applications**, then **Create App Integration**.
+1. Select the application type:
 
-## Plugin Configuration
+    1. Under **Sign-in method**, select **OIDC - OpenID Connect**.
+    1. Under **Application Type**, select **Web Application**.
 
-Add a plugin with the configuration below to your Route using an HTTP client or [Kong Manager][enable-plugin].
+1. Select **Next**. Configure the application:
+    1. Create a unique name for your application.
+    1. Under **Grant Type**, select **Authorization Code**.
+    1. In both the **Sign-in redirect URIs** and
+    **Sign-out redirect URIs** fields, enter a location handled by your Route
+    in Kong Gateway.
+
+        For this example, you can enter `https://kong.com/api`.
+
+    1. In the Assigments section, for **Controlled access**, choose your
+    preferred access level for this application. This preferred access level sets the permissions for
+    Okta admins.
+
+1. Save your settings to generate connection details.
+
+    Leave this page open. You'll need the details here to configure the Kong OIDC plugin.
+
+1. Add an Authorization Server. From the left sidebar, go to
+**Security > API > Authorization Server** and create a server named
+ **Kong API Management** with an audience and description. Click **Save**.
+
+    On the page that appears, note the **Issuer** address. You need this address
+    to configure the Kong OIDC Plugin.
+
+### Configure the OIDC plugin in {{site.base_gateway}}
+
+The following are steps you perform in {{site.base_gateway}} to enable the OIDC Plugin using Okta as an Identity Provider for the Authorization Code flow.
```

Comment:
Do we need this intro para at all? It only repeats what seems like obvious information at this point.